### DISCUSSION TITLE
Has anyone created an STL for the Luxonis Oak D Lite mount ?

### DISCUSSION THREAD

**Discussion by jerrynsr** [iRobotEducation/create3_docs#121] [Questions] [answered]
Hi,
  I purchased a Luxonis Oak D Lite before there are any supply chain issues, LOL.  I figured once college starts maybe the supply will be gobbled up.  I would like to add it to my Create3 to generate point cloud data.  Has anyone created an STL for the Luxonis Oak D Lite mount ?

**Comment by shamlian** [comment, marked as answer]
[Somebody did!](https://github.com/iRobotEducation/create3_docs/blob/main/docs/hw/data/brackets/C3-OAK-D-Lite-Mount.stl)

**Comment by jrhanobik** [comment]
[OAK-D Lite Mount.zip](https://github.com/iRobotEducation/create3_docs/files/9282041/OAK-D.Lite.Mount.zip)
I found and used this one and it works well.  Link here -> https://github.com/iRobotEducation/create3_docs/tree/main/docs/hw/data/brackets

**Reply by shamlian** [reply]
That's the same file as posted above. Glad you found it! :)
